core/drivers/clk/sam/at91_generated.c

```diff
@@ -140,6 +151,7 @@ struct clk *
 at91_clk_register_generated(struct pmc_data *pmc,
 			    const struct clk_pcr_layout *layout,
 			    const char *name, struct clk **parents,
+			    uint32_t *mux_table,
```

Comment:
Added `mux_table_size` variable after `uint32_t *mux_table` in the struct and use it to avoid accessing beyond `gck->mux_table`.

The `mux_table` is used for sama7g5 support as the clock source of [`sama7g5_gck`](https://github.com/TonyHan11/optee_os/commit/3d079675830ef1bd3ef7f8570d616ac5c60782f2#diff-63417ef6df5d25dad4762f9dbeca7887c5f97dd3c69a02ed6fc40f88261fafacR517) are from different PLL. For sama5d2 the sources are fixed and no mux is used.